src/components/ButtonPanel.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div id="btnPanel">
+      <div className="rows">
+        <Button btnName="AC" />
+        <Button btnName="+/-" />
+        <Button btnName="%" />
+        <Button btnName="÷" />
+      </div>
+
+      <div className="rows">
+        <Button btnName="7" />
+        <Button btnName="8" />
+        <Button btnName="9" />
+        <Button btnName="X" />
+      </div>
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.